modules/consensus/synchronize.go

```diff
@@ -10,22 +14,43 @@ import (
 	"github.com/NebulousLabs/bolt"
 )
 
+const (
+	// minNumOutbound is the minimum number of outbound peers required before ibd
+	// is confident we are synced.
+	minNumOutbound = 4
```

Comment:
I would like to bump this to 5.

I would also like to have logic to boot peers who are not contributing new blocks.